components/infieldbutton/index.css

```diff
@@ -123,28 +103,8 @@
 	&:active {
 		--mod-infield-button-background-color: var(--mod-infield-button-background-color-down, var(--spectrum-infield-button-background-color-down));
 		--mod-infield-button-icon-color: var(--mod-infield-button-icon-color-down, var(--spectrum-infield-button-icon-color-down));
-	}
-
-	&:focus-visible {
-		--mod-infield-button-background-color: var(--mod-infield-button-background-color-key-focus, var(--spectrum-infield-button-background-color-key-focus));
-		--mod-infield-button-icon-color: var(--mod-infield-button-icon-color-key-focus, var(--spectrum-infield-button-icon-color-key-focus));
-	}
-}
-
-@media (forced-colors: active) {
-	.spectrum-InfieldButton {
-		--highcontrast-infield-button-border-color: ButtonText;
-		--highcontrast-infield-button-border-color-active: Highlight;
-
-		&:disabled {
-			--highcontrast-infield-button-border-color: inherit;
-		}
 
-		&:not(:disabled):hover,
-		&:not(:disabled):active,
-		&:not(:disabled):focus-visible {
-			--highcontrast-infield-button-border-color: var(--highcontrast-infield-button-border-color-active);
-		}
+		transform: perspective(var(--spectrum-infield-button-downstate-perspective)) translateZ(var(--spectrum-component-size-difference-down));
```

Comment:
Again, not technically blocking, just a nitpick to be consistent: could we move this transform into the style definitions below, instead of with the token definitions up here? I think it should be a simple swap: 
```
// @ line 111
.spectrum-InfieldButton {
... // keep the border-style and all the stuff that's already here

    &:active {
        transform: perspective(var(--spectrum-infield-button-downstate-perspective)) translateZ(var(--spectrum-component-size-difference-down));
    }
}
```